.github/workflows/integration-tests.yml

```diff
@@ -136,6 +136,7 @@ jobs:
 
   eth-smoke-tests-matrix:
     environment: integration
+    if: false ## TODO: Debug
```

Comment:
@kalverra do we want to keep this TODO ?

Reply:
Still testing the workflow, will remove once I've validated the new test in CI